src/views/AddLead/index.js

```diff
@@ -178,6 +199,7 @@ class AddLead extends React.PureComponent {
                 source: [requiredCondition],
                 authors: [],
                 confidentiality: [requiredCondition],
+                priority: [requiredCondition],
```

Comment:
Priority is required here but not required in Browser Extension

Reply:
@AdityaKhatri is this required?

Reply:
It is not in the backend, but lets enforce it.
I've made necessary changes in client as well.